### PR TITLE
feat(Icon): Adding missing file icons

### PR DESCRIPTION
**Pull request by luzhon**
Adding the following missing file icons...
image

### CODE REVIEW

File: packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-files-indesign.tsx [hunk 1]
```diff
@@ -11,4 +11,5 @@ export default {
     </svg>
   ),
   styles: {},
+  exportedAs: 'filesIndesign',
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-files-indesign.tsx]: `files-indesign`?

File: packages/react/src/themes/teams/components/Icon/svg/icons/filesCode.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+        <path d="M22.2 6h-8c-.5 0-.9.2-1.3.5l-4.4 5c-.3.4-.5.8-.5 1.3v11.4c0 1 .8 1.8 1.8 1.8h12.5c1 0 1.8-.8 1.8-1.8V7.8c-.1-1-.9-1.8-1.9-1.8zM13 8v4H9.5L13 8zm10 16.2c0 .4-.3.8-.8.8H9.8c-.5 0-.8-.3-.8-.8V13h5V7h8.2c.5 0 .8.3.8.8v16.4zm-3.9-8c-.2-.2-.5-.2-.7-.1-.2.2-.2.5-.1.7l1.3 1.7-1.3 1.7c-.2.2-.1.5.1.7.1.1.2.1.3.1.1 0 .3-.1.4-.2l1.6-2c.1-.2.1-.5 0-.6l-1.6-2zm-5.6-.1c-.2-.2-.5-.1-.7.1l-1.6 2c-.1.2-.1.5 0 .6l1.6 2c.1.1.2.2.4.2.1 0 .2 0 .3-.1.2-.2.2-.5.1-.7l-1.3-1.7 1.3-1.7c.2-.2.1-.5-.1-.7zm3.9-2.1c-.3-.1-.5.1-.6.3l-2.5 8c-.1.3.1.5.3.6h.2c.2 0 .4-.1.5-.3l2.5-8c0-.2-.2-.5-.4-.6z" />
```
Comment (codepretty) [inline packages/react/src/themes/teams/components/Icon/svg/icons/filesCode.tsx]: remove this path. it's a remnant from the old way we had icons and isn't needed for this.

File: packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-files-indesign.tsx (hunk 1 above)
Comment (luzhon) [reply packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-files-indesign.tsx]: fixed!

File: packages/react/src/themes/teams/components/Icon/svg/icons/filesCode.tsx (hunk 2 above)
Comment (luzhon) [reply packages/react/src/themes/teams/components/Icon/svg/icons/filesCode.tsx]: fixed! thanks!

**Review by codepretty — APPROVED** [review]

**Pull request merged**